CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+# Master
+
+## Breaking
+
+- `findRecursive` now returns an array instead of an `ArrayElement`.
```

Comment:
Actually, I realise this may be problematic as you will loose access to the convenience element searching methods.